### DISCUSSION TITLE
Username and Password

### DISCUSSION THREAD

**Discussion by baconcow** [advplyr/audiobookshelf#1299] [Q&A]
Just went to connect to to the server within the iOS application and it is asking me for a Username and Password. I do not recall setting one up with Docker. What account is this for?

**Comment by DieselTech** [comment]
You wouldn't setup a username and password within docker. Once you loaded ABS for the first time it would ask you to create the root user. Once your in ABS as root you can make other user accounts from there.

**Comment by baconcow** [comment]
That makes sense. I am only at the point where I have run the Docker install command (on Windows 10) and have it running within Docker Desktop.

Is there any documentation of the next steps for running ABS for the first time? I have not been able to find any within the website's guides or this GitHub's documentation. I'm probably just overlooking it.

Thanks.


**Edit:**
1. I put the commands into the Run within Docker to start it instead of within the Windows 10 Command application (I might have previously put in the commands incorrectly).
2. I then went to the site at http://localhost:13378/.
3. Set up my password on first run.

**Reply by hobesman** [reply]
I don't know if it's actually stated anywhere (should be if it isn't) but in the meantime, if you access the server from a browser the first time, it prompts you to set up the initial account. Most of the administration is through the browser (for now). Then you can access it from mobile for listening.
